guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -197,6 +258,20 @@ If `eth0` was renamed to `em0`, call the installer to use the new interface name
 # leapp upgrade
 ----
 
+ifdef::satellite[]
++
+. If you run {Project} in a disconnected environment, add the `--no-rhsm` and `--enablerepo` parameters:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# leapp upgrade \
+--no-rhsm \
+--enablerepo BaseOS \
+--enablerepo AppStream \
+--enablerepo {RepoRHEL8ServerSatelliteServerProductVersion} \
+--enablerepo {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+----
+
```

Comment:
```suggestion
endif::[]

```
@evgeni you are indeed correct about the wrong alignment of the sections. I discovered this missing endif that was causing the issue.

Reply:
Indeed, this `ifdef` is not properly closed, good find!